src/moscot/base/problems/problem.py

```diff
@@ -270,10 +270,10 @@ def prepare(
         self._x = self._y = self._xy = self._solution = None
         # TODO(michalk8): in the future, have a better dispatch
         # fmt: off
-        if xy is not None and x is None and y is None:
+        if xy and not x and not y:
```

Comment:
Please change the typehints in the function definition - the `xy/x/y` can no longer be optional.

Reply:
I meant the type-hint in the definition, below they can't, since we do, e.g., `**xy`.